### DISCUSSION TITLE
Erors with doors 1.4.0.4

### DISCUSSION THREAD

**Discussion by FraxinusErit** [Pryaxis/TShock#1916] [General]
I have been having issues with my doors in my Tshock server. Often, doors will just disappear, and other NPCs won't move in because the houses are invalidated. Do you know what my problem might be?It happens to me as well as my friends playing the server.

**Comment by kubedzero** [comment]
This is being tracked in https://github.com/Pryaxis/TShock/issues/1774.

One solution noted in there by @hakusaro is if you give people the permission `tshock.ignore.sendtilesquare` it will work, but the damage will be that anyone can do anything to the world, essentially. Hack/cheat protection won't really work then

**Reply by FraxinusErit** [reply]
ok, thank you! Only a couple of close friends are on the server, so I would probably be okay with that. The server is more for them then for me anyways, I just wanted to do something nice for them.
thanks for the tip!
